examples/bakdata/atm-fraud-detection/config.yaml

```diff
@@ -12,4 +12,4 @@ kafka_rest_host: "http://localhost:8082"
 
 kafka_connect_host: "http://localhost:8083"
 
-defaults_path: ./defaults
+defaults_path: examples/bakdata/atm-fraud-detection/defaults
```

Comment:
it would be better if we could use relative paths here. otherwise, the path depends on where kpops is executed from

Reply:
What would the path be relative to?